src/commons/application/actions/__tests__/SessionActions.ts

```diff
@@ -161,6 +163,43 @@ test('setUser generates correct action object', () => {
   });
 });
 
+test('setGitHubOctokitInstance generates correct action object', async () => {
+  const authToken = 'testAuthToken12345';
+  const action = setGitHubOctokitInstance(authToken);
+
+  expect(action.type).toEqual(SET_GITHUB_OCTOKIT_INSTANCE);
+
+  // We need to do this because the auth() function returns a unknown Promise
+  // Typescript will not allow us to directly call the properties
+  action.payload.auth().then(authObject => {
+    const stringified = JSON.stringify(authObject);
+    const json = JSON.parse(stringified);
+
+    const keys = Object.keys(json);
+    const values = Object.values(json);
+
+    expect(keys.length).toEqual(values.length);
+
+    let existsKeyToken = false;
+    let existsKeyTokenType = false;
+
+    for (let i = 0; i < keys.length; i++) {
+      if (keys[i] === 'token') {
+        existsKeyToken = true;
+        expect(values[i] === authToken);
+      }
+
+      if (keys[i] === 'tokenType') {
+        existsKeyTokenType = true;
+        expect(values[i] === 'oauth');
+      }
+    }
```

Comment:
Why not `expect(authObject.token).toEqual('token')` etc?

If TypeScript is giving an issue, just cast it to `any`.. This is a valid use for that.